src/bundle/Resources/public/js/scripts/fieldType/ezdate.js

```diff
@@ -73,10 +73,13 @@
             return;
         }
 
+        const selectedDateWithUserTimezone = eZ.helpers.timezone.convertDateToTimezone(date[0]);
+
         date = new Date(date[0]);
         date = new Date(Date.UTC(date.getFullYear(), date.getMonth(), date.getDate()));
+        const microTime = date.getTime() - selectedDateWithUserTimezone.utcOffset() * 60 * 1000;
```

Comment:
Im not sure if this is a micro time. This is with the precision of milliseconds and microtime is smaller?

Reply:
yes, you are right, I changed to milliseconds